appviews/src/MenuBar/states/connected/PauseDowntimeModal.tsx

```diff
@@ -13,7 +13,7 @@ const PauseDowntimeModal: React.FC<Props> = ({ open, emit, dispatch }) => (
   <>
     <div
       className={cx(
-        `bg-white/50 dark:bg-white/10 backdrop-blur shadow-lg absolute z-20 px-3 py-2 rounded-xl w-80 transition-[opacity,transform] duration-300`,
+        `bg-white/50 dark:bg-white/10 shadow-lg absolute z-20 px-3 py-2 rounded-xl w-80 transition-[opacity,transform] duration-300`,
```

Comment:
this class was causing the bug shown here in catalina. i couldn't see any visible effect from removing it either, so hopefull it's ok.

![Uploading Screenshot 2024-11-14 at 12.48.58 PM.png…]()